src/pet.js

```diff
@@ -13,36 +13,39 @@ class Pet {
         const MAX_HUNGER = 10;
         const MIN_FITNESS = 0;
 
-        return this.age < MAX_AGE && this.hunger < MAX_HUNGER && this.fitness > MIN_FITNESS;
+        return this.age <= MAX_AGE && this.hunger < MAX_HUNGER && this.fitness > MIN_FITNESS;
     }
 
     get deadMessage() {
-        const DEAD_MESSAGE = 'Your pet is no longer alive :(';
+        return 'Your pet is no longer alive :(';
+    }
 
-        return DEAD_MESSAGE;
+    isAliveCheck() {
+        if (!this.isAlive) {
+            throw new Error(this.deadMessage);
+        }
+        return;
```

Comment:
nice, if you didn't write return here, would the function still work as expected?